src/standalone/Soil/boundary_conditions.jl

```diff
@@ -418,21 +457,27 @@ function ClimaLand.boundary_flux(
 )::ClimaCore.Fields.Field
     FT = eltype(Δz)
     # Approximate K_bc ≈ K_c, ψ_bc ≈ ψ_c (center closest to the boundary)
-    p_len = Spaces.nlevels(axes(p.soil.K))
-    # We need to project the center values onto the face space.
-    K_c = Fields.Field(
-        Fields.field_values(Fields.level(p.soil.K, p_len)),
-        axes(Δz),
-    )
-    ψ_c = Fields.Field(
-        Fields.field_values(Fields.level(p.soil.ψ, p_len)),
-        axes(Δz),
-    )
+    surface_space = axes(Δz)
+    K_c = get_top_surface_field(p.soil.K, surface_space)
+    ψ_c = get_top_surface_field(p.soil.ψ, surface_space)
 
     # Calculate pressure head using boundary condition
+    # Extract parameters; if these are fields, get the values at the surface
     (; hydrology_cm, θ_r, ν, S_s) = model.parameters
+    if typeof(hydrology_cm) <: ClimaCore.Fields.Field
+        hcm_bc = get_top_surface_field(hydrology_cm, surface_space)
```

Comment:
`get_top_surface_field` could be get a method for scalars that just returns itself so that you can remove the if here.

Reply:
thanks! did so